x.py

```diff
@@ -170,9 +183,13 @@ def generate_stackbrew_library():
                 tags.append(version_tag)
             tags.append("latest")
 
+        arches = debian_arches
```

Comment:
When I suggested this line, I forgot it wasn't copying `debian_arches`, only creating a reference and then mutating it. It caused `ppc64le` to be added twice. To fix this, we just need to copy it.
```suggestion
        arches = debian_arches[:]
```

Reply:
Thank you, it is fixed now.